<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentTranslationDialog</class>
 <widget class="QDialog" name="DocumentTranslationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate Document</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fileGroup">
     <property name="title">
      <string>Document</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="inputFileLabel">
        <property name="text">
         <string>document.docx</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Output:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="outputLayout">
        <item>
         <widget class="QLineEdit" name="outputFileEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="aiStatusLabel">
     <property name="text">
      <string>AI Improvement: Disabled</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="progressGroup">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="progressLayout">
      <item>
       <widget class="QLabel" name="translationProgressLabel">
        <property name="text">
         <string>Translation: Ready</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="translationProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="llmProgressLabel">
        <property name="text">
         <string>AI Improvement: Ready</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="llmProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start Translation</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
